frontend/README.md

```diff
@@ -92,14 +91,14 @@ cd packages/employee-frontend
 ICONS=free yarn dev
 ```
 
-### Using pro icons
+### Using professional icons
```

Comment:
Please fix all the links pointing to this title in:
```
compose/README.md:With [pro icons](../frontend/README.md#using-pro-icons)
frontend/packages/employee-frontend/README.md:so you need to [configure that first](../README.md#using-pro-icons)
frontend/packages/enduser-frontend/README.md:so you need to [configure that first](../README.md#using-pro-icons)
```

Reply:
I think they're OK now.